tests/operators/test_agnostic_load_file.py

```diff
@@ -270,6 +271,39 @@ def test_aql_local_file_to_postgres_no_table_name(self):
             "taxes": 3167.0,
         }
 
+    def test_aql_local_file_to_sqlite_no_table_name(self):
```

Comment:
We could avoid these new 22 lines of code if we extended the `test_load_file` method to also validate `sqlite`:
https://github.com/astro-projects/astro/pull/119/files#diff-811d2b3637fa84cdde5a2e2ab5c85d15b16b80bf39f845dc082913cda5a5ea57R559

It already supports all the other DBs!

Reply:
I was thinking of the same, but the queries passed to different DB are not same, we probably need to put that in a fixture as well.

Reply:
How different are they?